libvirt/config_libvirt.sh

```diff
@@ -13,20 +13,27 @@ ARCH=$(uname -m)
 TARGET_ARCH=${ARCH/x86_64/amd64}
 
 installGolang() {
-    export PATH=$PATH:/usr/local/go/bin
+    export PATH=/usr/local/go/bin:$PATH
     export GOROOT=/usr/local/go
     export GOPATH=$HOME/go
+    REQUIRED_GO_VERSION="$(yq '.tools.golang' versions.yaml)"
     if ! command -v "yq" >/dev/null; then
         echo "Installing latest yq"
         sudo wget https://github.com/mikefarah/yq/releases/latest/download/yq_linux_${TARGET_ARCH} -O /usr/bin/yq && sudo chmod a+x /usr/bin/yq
     fi
     if [[ -d /usr/local/go ]]; then
-        echo "go is installed, set path."
+        installed_go_version=$(go version | awk '{print $3}')
+        if [[ "$(printf '%s\n' "$REQUIRED_GO_VERSION" "$installed_go_version" | sort -V | head -1)" != "$REQUIRED_GO_VERSION" ]]; then
```

Comment:
`go version | awk '{print $3}'` includes `go` e.g. `go1.18.10`. So I think we either need to strip this, or add the extra "go" into the check?
```suggestion
        if [[ "$(printf '%s\n' "go$REQUIRED_GO_VERSION" "$installed_go_version" | sort -V | head -1)" != "$REQUIRED_GO_VERSION" ]]; then
```

Reply:
On reflection probably I think adding `go` as my suggestion isn't the best approach and stripping `go` out of the version which something like `installed_go_version=$(v=$(go version | awk '{print $3}') && echo ${v#go})`. I'm sure there is a nicer way to do that though!

Reply:
```
installed_go_version=$(v=$(go version | awk '{print $3}') && echo ${v#go})
```
works well! Thanks @stevenhorsman